Resources/public/javascript/forms.js

```diff
@@ -1,16 +1,61 @@
+let removePopup = `<div class="sonata-ba-delete popup-file-delete mfp-hide">
+                        <div class="box box-danger">
+                            <div class="box-header">
+                                <h3 class="box-title">Bevestig verwijdering</h3>
+                            </div>
+                            <div class="box-body">
+                                Weet je zeker dat de geselecteerde afbeelding moet worden verwijderd?
+                            </div>
+                            <div class="box-footer clearfix">
+                                <button class="btn btn-danger js-trigger-save"><i class="fa fa-trash" aria-hidden="true"></i> Ja, verwijder</button>
+                                <button class="btn btn-success js-trigger-cancel" href="/admin/wearejust/example/post/1/edit">
+                                <i class="fa fa-pencil" aria-hidden="true"></i>
+                                Nee, annuleer</button>
+                            </div>
+                        </div>
+                        <button title="Close (Esc)" type="button" class="mfp-close js-trigger-cancel">×</button>
+                        <div class="anim-loading">
+                            <label></label>
+                            <label></label>
+                            <label></label>
+                            <label></label>
+                        </div>
+                    </div>`;
+
 class imageRemove {
     constructor(item) {
+        $('body').append(removePopup);
         this.item = item
         this.item.find('input[type="checkbox"]').on('change', this.change.bind(this));
+        $('.js-trigger-save').on('click', forceSave);
```

Comment:
.append() can have a small delay, which causes the selectors after it to not work, _sometimes_. Wrap the removePopup in a jQuery selector before appending it and use .find() after.